warp10/src/main/java/io/warp10/fdb/FDBPool.java

```diff
@@ -60,6 +60,14 @@ public synchronized Database getDatabase() {
     return db;
   }
 
+  public double[] getDatabasesBusyness() {
+      double[] busyness = new double[databases.length];
```

Comment:
Instead of returning a newly allocated array, it would be better to modify the method so it updates the Sensision metrics. In order to speed things up (the call to Sensision.set being synchronized), you could keep track of the last update and only update the metrics if the last update was more than 1s ago, since the underlying FDB metrics are not updated more frequently than that it would be sufficient to have an always up to date value.

Reply:
Updated accordingly.